src/models/async/Summary.ts

```diff
@@ -0,0 +1,4 @@
+export interface Summary {
```

Comment:
there are a few types that have no comments

Reply:
Actually almost all of the types in this folder have no comments